cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -0,0 +1,119 @@
+                    'custom_is_part_of_ctc':component.is_part_of_ctc,
+                    'custom_is_arrear':component.is_arrear,
+                    'custom_component':component.component,
+                    
+
+                    
+                })
+                doc1.insert()
+                frappe.db.commit()  
+
+            else:  
+                custom_component = frappe.get_doc("salary Component", get_component[0].name)
```

Ruff (0.8.2)

Local variable `custom_component` is assigned to but never used

Remove assignment to unused variable `custom_component`

(F841)